### DISCUSSION TITLE
How to include scanning only for a single iBeacon?

### DISCUSSION THREAD

**Discussion by rwjack** [ESPresense/ESPresense#1062] [Support]
I'm guessing the first field is what I'm looking for:

![image](https://github.com/ESPresense/ESPresense/assets/59068073/b3c2c9eb-9eb6-4aec-a577-9865fc7ec36c)

But I don't understand what data do I need to put there?

Here's my HA sensor config:
```
- platform: mqtt_room
  name: "Beacon 1"
  device_id: "iBeacon:AAAAAAAA-1212-B2B3-6E2E-ABCABC000001-1234-4321" # I redacted this
  state_topic: "espresense/rooms"
  away_timeout: 60
```

If I put `iBeacon:AAAAAAAA-1212-B2B3-6E2E-ABCABC000001-1234-4321` in the field, then none of my beacons are detected, including this one that I want to be detected.

**Comment by DTTerastar** [comment]
It's case sensitive.  Make sure to copy and paste and it should work.
